pkg/process/checks/process_nix.go

```diff
@@ -89,3 +92,27 @@ func calculatePct(deltaProc, deltaTime, numCPU float64) float32 {
 	pct = math.Max(pct, 0.0)
 	return float32(pct)
 }
+
+// warnECSLinuxFargateMisconfig pidMode is currently not supported on fargate windows, see docs: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#task_definition_pidmode
+func warnECSFargateMisconfig(containers []*model.Container) {
+	if fargate.IsFargateInstance() && !isECSFargatePidModeSetToTask(containers) {
```

Comment:
You can just remove this fargate check since your other function checks it.

Reply:
Actually can't remove this because then non-Fargate envs will emit the log